x-pack/plugins/observability_solution/dataset_quality/server/utils/to_boolean.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function toBoolean(value: any): boolean {
```

Comment:
Could you explain me some usage of this util? I don't get it, maybe I'm missing something. Why value is any? 👀

Reply:
if you see the response received from elasticsearch, boolean is returned as string. Hence i created this utility which will return the correct value for a string boolean.

i can change the input type from any to string | boolean

<img width="703" alt="image" src="https://github.com/user-attachments/assets/a5a39021-a2e9-431e-a2ee-dd5425b3335f">

Reply:
yes please, better to have it like that